### DISCUSSION TITLE
Let Copilot look up.

### DISCUSSION THREAD

**Discussion by dezren39** [community/community#12491] [Copilot Conversations]
I have noticed that copilot doesn't 'get' things beneath the cursor, except maybe sometimes? If possible, I wish it could see code and comments beneath as well, and use that contextually. Maybe above is slightly preferenced over below for consistency?

**Comment by hsenag** [comment]
Hi, thanks for the comment. We are trying out making Copilot look at the code after the cursor - there's some description of the underlying functionality in [this blogpost from OpenAI](https://openai.com/blog/gpt-3-edit-insert/) - but at present I don't know when that will be rolled out to all users.
